.github/workflows/runnable_cxx.yml

```diff
@@ -57,106 +50,65 @@ jobs:
-          - { os: windows-2019, target: msvc-2015 }
-          - { os: windows-2019, target: msvc-2013 }
-          - { os: windows-2019, target: clang-13.0.0 }
-          - { os: windows-2019, target: clang-12.0.0 }
-          - { os: windows-2019, target: clang-11.0.0 }
-          - { os: windows-2019, target: clang-10.0.0 }
-          - { os: windows-2019, target: clang-9.0.0 }
-          - { os: windows-2019, target: clang-8.0.0 }
-
-        # This sets the configuration for each jobs
-        # There's a bit of duplication involved (e.g. breaking down g++-9.3 into 2 strings),
-        # but some items are unique (e.g. clang-9.0.0 and 4.0.1 have differences in their naming).
         include:
-          # Clang boilerplate
-          - { target: clang-13.0.0, compiler: clang, cxx-version: 13.0.0 }
-          - { target: clang-12.0.0, compiler: clang, cxx-version: 12.0.0 }
-          - { target: clang-11.0.0, compiler: clang, cxx-version: 11.0.0 }
-          - { target: clang-10.0.0, compiler: clang, cxx-version: 10.0.0 }
-          - { target: clang-9.0.0, compiler: clang, cxx-version: 9.0.0 }
-          - { target: clang-8.0.0, compiler: clang, cxx-version: 8.0.0 }
-          # g++ boilerplace
-          - { target: g++-11, compiler: g++, cxx-version: 11.2.0, major: 11 }
-          - { target: g++-10, compiler: g++, cxx-version: 10.3.0, major: 10 }
-          - { target: g++-9, compiler: g++, cxx-version: 9.4.0, major: 9 }
-          # Platform boilerplate
-          - { os: ubuntu-22.04, arch: x86_64-linux-gnu-ubuntu-20.04 }
-          - { os: macos-13,  arch: x86_64-apple-darwin }
-          # Clang 9.0.0 have a different arch for OSX
-          - { os: macos-13, target: clang-9.0.0, arch: x86_64-darwin-apple }
-          # Those targets will generate artifacts that can be used by other testers
-          - { storeArtifacts: false }
-          - { os: ubuntu-22.04, target: g++-9,    storeArtifacts: true }
-          - { os: macos-13,  target: clang-9.0.0, storeArtifacts: true }
-          #- { os: windows-2019, target: msvc-2019,   storeArtifacts: true }
+          # Linux, clang:
+          # NOTE: cannot test MODEL=32 with clang on Linux, due to a single failure as of April 2025:
+          #       runnable_cxx/cppa.d:261: cppa.check13956: Assertion `arg6 == 6' failed.
+          - { os: ubuntu-24.04, compiler: clang-18, model: 64 }
+          - { os: ubuntu-22.04, compiler: clang-15, model: 64 }
+          - { os: ubuntu-24.04, compiler: clang-14, model: 64 }
+          - { os: ubuntu-22.04, compiler: clang-11, model: 64 }
+          # Linux, g++:
+          - { os: ubuntu-24.04, compiler: g++-13, model: 64 }
+          - { os: ubuntu-24.04, compiler: g++-13, model: 32 }
+          - { os: ubuntu-22.04, compiler: g++-12, model: 64 }
+          - { os: ubuntu-22.04, compiler: g++-12, model: 32 }
+          - { os: ubuntu-22.04, compiler: g++-9,  model: 64 }
+          - { os: ubuntu-22.04, compiler: g++-9,  model: 32 }
```

Comment:
`g++-9` is the oldest available version on both Ubuntu 22 and 24. I went with Ubuntu 22.